packages/bpk-component-breadcrumb/readme.md

```diff
@@ -0,0 +1,50 @@
+# bpk-component-breadcrumb
+
+> Backpack breadcrumb component.
+
+## Installation
+
+```sh
+npm install bpk-component-breadcrumb --save-dev
+```
+
+## Usage
+```js
+import React, { Component } from 'react';
+import BpkBreadcrumb, { BpkBreadcrumbItem } 'bpk-component-breadcrumb';
+
+
+export default class App extends Component {
+  render() {
+    return (
+      <BpkBreadcrumb label="Default breadcrumb">
+        <BpkBreadcrumbItem href="/">Home</BpkBreadcrumbItem>
+        <BpkBreadcrumbItem href="/page-1">Page 1</BpkBreadcrumbItem>
+        <BpkBreadcrumbItem active aria-current="page">
```

Comment:
I think `active` should set `aria-current="page"` unless the consumer has supplied it themselves

Reply:
aria-current can have a value other than 'page'. see https://tink.uk/using-the-aria-current-attribute/

Reply:
Hugo's suggestion would support both, so if the user doesn't supply `aria-current`, but they do set `active={true}`, `BpkBreadcrumbItem` would automatically add `aria-current="page"` to the final output.

However if the user does supply `aria-current`, `BpkBreadcrumbItem` uses that instead, regardless of the value of `active`.

Reply:
What Shaun said 👍 From my reading it seems like in 99% of cases `"page"` is going to be what you want and setting it by default on `active` items thus seems like a good idea